src/constants/common.ts

```diff
@@ -229,7 +230,9 @@ export const getFileSitemap = (filePath: string): string | null => {
 
   const file = fs.readFileSync(filePath, 'utf8');
   const isLocalSitemap = isSitemapContent(file);
-  return isLocalSitemap ? filePath : null;
+  let isLocalFileOrSitemap = isLocalSitemap ? filePath : null;
```

Comment:
Why is there a duplicate `isLocalSitemap` and `isLocalFileOrSitemap`? Seems unnecessary